### DISCUSSION TITLE
[Dashboard]

### DISCUSSION THREAD

**Discussion by thawmisntme4081** [thawmisntme4081/toodle#15] [Ideas]
# Admin

# Teacher

# Staff

# Student

# Parent

**Comment by DuySeu** [comment]
![image](https://github.com/user-attachments/assets/4e0c0552-5a93-4795-ab73-0e8913e2c8ce)
![image](https://github.com/user-attachments/assets/6fafde89-fdfe-4de7-a705-b3a1c956b896)
![image](https://github.com/user-attachments/assets/df3a8640-a3e9-43d9-88c3-6d38fec0c7d2)

**Reply by DuySeu** [reply]
Dashboard

**Comment by DuySeu** [comment]
![image](https://github.com/user-attachments/assets/c7f8d745-eaa9-49d7-8239-b3baac74ff8e)

**Comment by DuySeu** [comment]
![image](https://github.com/user-attachments/assets/db107685-280f-4003-8e78-2dce59ee6c7f)

**Comment by DuySeu** [comment]
![image](https://github.com/user-attachments/assets/3df23bcf-0bdf-4506-9a97-2981c2b0f957)

**Comment by DuySeu** [comment]
![image](https://github.com/user-attachments/assets/2288b8ed-cf59-4554-96c2-906d45379358)

**Comment by DuySeu** [comment]
![image](https://github.com/user-attachments/assets/05c76ccc-4b67-417b-896d-bf023ba97b7f)

**Comment by DuySeu** [comment]
![image](https://github.com/user-attachments/assets/386efee4-3a87-4dcf-995e-a9a6d0a86c8c)
Student
